content/help/ops/telemetry/missing-metrics/index.md

```diff
@@ -0,0 +1,203 @@
+
+## Verify the Mixer post-install job completed successfully
+
+A job running after the initial Istio installation completes supplies the default configuration for Mixer. If it fails to complete, Mixer doesn’t receive any configuration. Thus, Mixer produces no telemetry including logs, metrics, and traces among others, for any report calls. To verify the job is running as intended, run the following command:
+
+{{< text bash >}}
+$ kubectl -n istio-system get job
+istio-cleanup-secrets          1         1            7d
+istio-grafana-post-install     1         1            7d
+istio-mixer-post-install-1.0   1         1            7d
+istio-security-post-install    1         1            7d
+{{< /text >}}
+
+If the `istio-mixer-post-install-1.0` job doesn’t complete successfully, please delete the `istio-mixer-post-install-1.0` job with the following
+command:
+
+{{< text bash >}}
+$ kubectl -n istio-system delete job istio-mixer-post-install-1.0
+{{< /text >}}
+
+Once the job is deleted, retry installing Istio.
+
+## Verify Mixer is receiving Report calls
+
+Mixer generates metrics to monitor its own behavior. The first step is to check these metrics:
+
+1. Establish a connection to the Mixer self-monitoring endpoint for the istio-telemetry deployment. In Kubernetes environments, execute the following command:
+
+    {{< text bash >}}
+    $ kubectl -n istio-system port-forward <istio-telemetry pod> 9093 &
+    {{< /text >}}
+
+1. Verify successful report calls. On the Mixer self-monitoring endpoint, search for `grpc_server_handled_total`. You should see something like:
+
+    {{< text plain >}}
+    grpc_server_handled_total{grpc_code="OK",grpc_method="Report",grpc_service="istio.mixer.v1.Mixer",grpc_type="unary"} 68
+    {{< /text >}}
+
+    If you do not see any data for `grpc_server_handled_total` with a `grpc_method="Report"`, then Envoy is not calling Mixer to report telemetry.
+
+1. In this case, ensure you integrated the services properly into the mesh. You can achieve this task with either [automatic or manual sidecar injection](/docs/setup/kubernetes/sidecar-injection/).
+
+## Verify the Mixer rules exist
+
+In Kubernetes environments, issue the following command:
+
+{{< text bash >}}
+$ kubectl get rules --all-namespaces
+NAMESPACE      NAME        AGE
+istio-system   kubeattrgenrulerule      13d
+istio-system   promhttp                 13d
+istio-system   promtcp                  13d
+istio-system   stdio                    13d
+istio-system   tcpkubeattrgenrulerule   13d
+{{< /text >}}
+
+If the output shows no rules named `promhttp` or `promtcp`, then the Mixer configuration for sending metric instances to the Prometheus adapter is missing. You must supply the configuration for rules connecting the Mixer metric instances to a Prometheus handler.
+
+For reference, please consult the [default rules for Prometheus]({{< github_file >}}/install/kubernetes/helm/istio/charts/mixer/templates/config.yaml#L592-L619).
```

Comment:
This link, and the other github links don't work. What's the correct link to use?

Note that using line numbers here isn't a great idea, since that breaks as soon as the target is updated.

Reply:
https://github.com/istio/istio/blob/release-1.0/install/kubernetes/helm/istio/charts/mixer/templates/config.yaml is the link to the artifact. If we want to avoid line #s, we should probably add text around searching for `promhttp` or whatever is appropriate to each link.